### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by isanidiot**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1103,6 +1155,35 @@ testers are expected to do more *exploratory* testing.
...
+1. Adding a description
+
+   2. Test case: `description 2 d/Eager to invest`<br>
+      Expected: Description of the second client changed to `Eager to invest`. 
+   
+   3. Test case: `description d/Eager to invest`<br>
+      Expected: No change in descriptions made. Error details shown in command display.
+  
+   3. Other test cases: `description`, `description 0 d/Eager to invest`<br>
+      Expected: No change in descriptions made. Error details shown in command display.
```
Comment (ReaganTan00) [inline docs/DeveloperGuide.md:1176]: The prefix should be ds/ instead of d/

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1103,6 +1155,35 @@ testers are expected to do more *exploratory* testing.
...
+   
+
+2. Removing a description
+
+   2. Test case: `description 2 d/`<br>
+      Expected: Description of the second client removed. "Removed Description: " message shown in command display. 
+   
+   3. Other test cases: `description`, `description 0 d/`<br>
+      Expected: No change in descriptions made. Error details shown in command display.
```
Comment (ReaganTan00) [inline docs/DeveloperGuide.md:1185]: Likewise here

**Review by ReaganTan00 — CHANGES_REQUESTED** [review]
Small changes to prefix and will be okay

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (isanidiot) [reply docs/DeveloperGuide.md:1176]: yep. was running the old version. thanks!

**Review by ReaganTan00 — APPROVED** [review]
lgtm

**Pull request merged**